pkg/connector/resource_types.go

```diff
@@ -10,12 +10,12 @@ var collaboratorResourceType = &v2.ResourceType{
 	Id:          "collaborator",
 	DisplayName: "Collaborator",
 	Traits:      []v2.ResourceType_Trait{v2.ResourceType_TRAIT_USER},
-	Annotations: annotations.New(&v2.SkipEntitlementsAndGrants{}),
 }
 
 var privilegeResourceType = &v2.ResourceType{
 	Id:          "privilege",
 	DisplayName: "Privilege",
+	Annotations: annotations.New(&v2.SkipGrants{}),
```

Comment:
I'm glad we added this annotation and that people are using it.

Ideally we should skip grants/entitlements based on whether these methods exist, but that would require some changes to interfaces. :/